### DISCUSSION TITLE
Broken download link in 4.0.1

### DISCUSSION THREAD

**Discussion by steeeffen** [coder/code-server#4767] [Q&A]
By using install.sh (of version 3.11), wget reports a 404:

Installing vhttps://github.com/coder/code-server/releases/tag/v4.0.1 deb package from GitHub releases.

+ mkdir -p ~/.cache/code-server
+ curl -#fL -o ~/.cache/code-server/code-server_https://github.com/coder/code-server/releases/tag/v4.0.1_amd64.deb.incomplete -C - https://github.com/cdr/code-server/releases/download/vhttps://github.com/coder/code-server/releases/tag/v4.0.1/code-server_https://github.com/coder/code-server/releases/tag/v4.0.1_amd64.deb
################################################################################################################################################## 100,0%#=O#-   #     #                                                                                                                                         
curl: (22) The requested URL returned error: 404

**Comment by jsjoeio** [comment]
It seems like the URL you are curling is incorrect? Or am I missing something?

**Comment by lg-nineoneninethreenine** [comment]
Oh, it's the `cdr` link, not the `coder` link. Can I fix this? In a PR of course.

**Reply by jsjoeio** [reply]
Yes please!
